github/resources.py

```diff
@@ -0,0 +1,132 @@
+import functools
+import io
+import json
+import os
+
+import pyjq
+
+
+results_dir = os.environ["RESULTS_DIR"]
+organization = os.environ["Organization"]
+today = os.environ["TODAY"]
```

Comment:
Lets move these env vars to cli flags that live in `conftest.py`. Something like `--github-organization` and `--github-data-directory`

Reply:
@ajvb What's our pattern here? env vars to pass into container and then restate as explicit options on command line?

Are option defaults allowed? (i.e. take from env if not supplied on command line?)

Reply:
`env vars to pass into container and then restate as explicit options on command line?` - yes!